src/design-system-team.md.njk

```diff
@@ -15,6 +15,7 @@ If you want to contact the team you can
 ## Members
 
 - Hanna Laakso – Frontend Developer
+- Joe Lanman - Lead Interaction Designer
```

Comment:
```suggestion
- Joe Lanman – Lead Interaction Designer
```

Reply:
Tiny thing but to keep things consistent we can use an endash rather than a hyphen here.